scripts/merge-dev.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+git checkout dev
+git merge main --no-ff
+
+CONFLICTS=$(git ls-files -u | wc -l)
```

Comment:
you can just check if `$?` which contains the status code of the last command) is 0. If it's something else, then you have an issue

Reply:
```bash
git checkout dev
git merge main --no-ff --no-commit
if [ "$?" -gt 0 ] ; then
   echo "There is a merge conflict. Aborting ..."
   git merge --abort
   exit 0
fi
git push dev
```

Reply:
the test failed , something wrong with the condition